keyboards/tradestation/keymaps/default/keymap.c

```diff
@@ -0,0 +1,61 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	LAYOUT_ortho_4x4(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+	LAYOUT_tradestatopm(
```

Comment:
Looks like you mistyped this:
```suggestion
 	LAYOUT_tradestation(
```